src/locale/en/index.json

```diff
@@ -130,5 +130,7 @@
     "actionable_message_section1_text": "This is an actionable message card section1, please note if there is only one potential Action tied to section then buttons won't display in the card.",
     "actionable_message_section2": "Actionable message section2",
     "actionable_message_section2_text": "This is an actionable message card. as this section has multiple potential actions in card, action button will be there to expand and hide the controls.",
+    "like_message":"You liked this message",
```

Comment:
"this message" -- which message? Should we just say "You liked a message"?

Reply:
Done.